PubnativeLite/PubnativeLite/Ad Request/VWAdFactory.m

```diff
@@ -135,4 +135,21 @@ - (void)setIDFA:(VWAdRequestModel *)adRequestModel {
     }
 }
 
+- (void)setContentCategoryIDsForAdRequestModel:(VWAdRequestModel *)adRequestModel withContentCategoryIDs:(NSMutableArray *)contentCategoryIDs {
+    NSMutableString *contentCategoryIDsString = [NSMutableString new];
+    if (contentCategoryIDs.count) {
+      [contentCategoryIDs enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
+        NSInteger catID = [obj integerValue] ?: 97;
+        if (idx == 0) {
+          [contentCategoryIDsString appendFormat:@"&c=%ld", (long)catID];
+        } else {
+          [contentCategoryIDsString appendFormat:@"%%2C%ld", (long)catID];
+        }
+      }];
+    } else { /* Default to "News and Information". */
+      [contentCategoryIDsString appendString:@"&c=97"];
```

Comment:
shouldnt we send empty request as a default ?

Reply:
I think by default there should be a value for the content category ID. In Verve SDK, they set it to 97.

Reply:
Yes @can-soykarafakili . This is right. The default category is NEWS which is 97